cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -275,10 +275,12 @@ angular.module(PKG.name + '.commons')
         }
       },
       'multi-select': {
-        element: '<my-multi-select-dropdown></my-multi-select-dropdown>',
+        element: '<multi-select></multi-select>',
         attributes: {
-          'ng-model': 'model',
-          'config': 'myconfig'
+          'value': 'model || myconfig.properties.defaultValue || [myconfig["widget-attributes"].defaultValue]',
```

Comment:
don't set the default value in here. Setting this to `model` should be enough

Reply:
Good to know, thanks!